query.js

```diff
@@ -664,6 +664,8 @@ function parseExport(res) {
         }))
 
         // travels
+        var travelIndexCount = 0;
+
         if (d.travels && d.travels.length) d.travels.forEach(a => travels.push({
```

Comment:
To make this code a bit more readable, I would suggest doing something like this: 

```suggestion
        if (d.travels && d.travels.length) d.travels.forEach((a, i) => travels.push({
```

This would make `i` equal to the index. Then you can set travelIndex to `i + 1`.